server/controllers/article.controller.js

```diff
@@ -300,5 +301,52 @@ class ArticleController {
       return Response(res, 200, 'Article found.', [articleObject]);
     }
   }
+
+  /**
+   * @description Get top rated articles controller
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} Response
+   */
+  static async getTopArticle(req, res) {
+    const allarticles = await Article.findAll({
+      where: {
+        published: true,
+      },
+      attributes: {
+        exclude: ['published', 'userId']
+      },
+      include: [{
+        model: User,
+        as: 'author',
+        attributes: {
+          exclude: ['istokenreset', 'password', 'social', 'roleId', 'subscribed', 'id']
+        }
+      }]
+    });
+    const rate = allarticles.map(async (article) => {
```

Comment:
Good job @davidokonji , kindly look into this variable naming and make it follow the right convention.